src/components/ActivitySearchbar.vue

```diff
@@ -106,6 +107,14 @@ export default {
         .child("count")
         .set(count + 1);
       this.suggested = "";
+    },
+    fuseSearch(options, search) {
+      const fuse = new Fuse(options, {
+        keys: ["name", "searchName", "keywords"]
+      });
+      return search.length
```

Comment:
Ahh! I had tried this once before but didn't figure out this simple way to show the whole list by default! Nice.